cedar-policy-core/src/parser/err.rs

```diff
@@ -685,6 +666,10 @@ pub fn expected_to_string(expected: &[String], config: &ExpectedTokenConfig) ->
 }
 
 /// Multiple parse errors.
+//
+// CAUTION: this type is publicly exported in `cedar-policy`.
+// Don't make fields `pub`, don't make breaking changes, and use caution when
+// adding public methods.
 #[derive(Clone, Debug, Default, PartialEq, Eq)]
 pub struct ParseErrors(pub Vec<ParseError>);
```

Comment:
"Don't make fields `pub`" is confusing when the type already has a pub field

Reply:
Maybe we should have this field be `pub(crate)` instead of `pub`, now that it's publically exported?  Or should this type be an exception?

Reply:
Changed to `pub(crate)` in the latest commit. I plan to make the field private in a future PR.

Reply:
On second thought... undid this in the latest commit since I don't want to fix the resulting build failures. I promise that I'll take care of it in a PR in the next few days!